app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -13,7 +13,7 @@ import edu.ucandroid.weathernotice.service.LocationService
 
 class MainViewModel : ViewModel() {
 
-    private lateinit var firestore : FirebaseFirestore
```

Comment:
Lateinit is unnecessary here, as this variable is guaranteed to be initialized via the constructor calls from FirebaseFirestore. 

https://kotlinlang.org/docs/properties.html#late-initialized-properties-and-variables

Reply:
Acknowledge.  I sure I the that did this, so thank you for the correction.